wiki/Contests/en.md

```diff
@@ -236,6 +236,18 @@ The **osu!trivium Quiz** is a trivia contest where players compete in answering
 
 *Unofficial contests* are contests that are hosted by the members of the community.
 
+### Mapping to Pander: Taiko
+
+**Mapping to Pander: Taiko** is an osu!taiko mapping contest series with two primary features: there is only one judge per iteration, and contestants may map any song.
+
+The goal this contest is simple: Contestants must create a map that panders best to the chosen judge for the contest. The judge will not be given any criteria; it will be up to them how they score and rank the entries, either by making their own criteria, or simply sorting them from best to worst.
```

Comment:
```suggestion
The goal this contest is simple: contestants must create a map that panders best to the chosen judge for the contest. The judge will not be given any criteria; it will be up to them how they score and rank the entries, either by making their own criteria, or simply sorting the maps from best to worst.
```